index.bs

```diff
@@ -106,7 +106,7 @@ There is currently no native CSS mechanism available to developers that achieves
                 Without the use of JavaScript, some browsers require that the contents of the <code>iframe</code> are stored in a <em>separate HTML document</em> rather than being part of the current document. This necessitates a new document composition paradigm that’s very different from the methods currently employed by developers.
             </li>
             <li>
-                In the majority of browsers, the contents of an <code>iframe</code> do not natively affect the dimensions of the <code>iframe</code> itself. The only known exception to this at the time of writing is iOS Safari. The contents of a container that is the subject of a Container Query are still expected to be able to affect the layout of its parents in the <em>opposite</em> dimension of the query. For example, in the case of a Container Query that is based on a containers <em>width</em>, it is still expected that the container’s contents affect the <em>height</em> of a container, subject to any overflow rules.
+                In the majority of browsers, the contents of an <code>iframe</code> do not natively affect the dimensions of the <code>iframe</code> itself. The only known exception to this at the time of writing is iOS Safari which sets the height of the <code>iframe</code> to the height of it's content. The contents of a container that is the subject of a Container Query are still expected to be able to affect the layout of its parents in the <em>opposite</em> dimension of the query. For example, in the case of a Container Query that is based on a containers <em>width</em>, it is still expected that the container’s contents affect the <em>height</em> of a container, subject to any overflow rules.
```

Comment:
See my comment on the old commit here https://github.com/WICG/cq-usecases/pull/64/files/28f7e75daa6f74948281a2b4a7ad26cb578cfe35#r168499068 – I don't know if it's worth mentioning Safari's non-standard behavior.